lib/common/grpc-service.js

```diff
@@ -266,6 +266,25 @@ GrpcService.prototype.request = function(protoOpts, reqOpts, callback) {
     request: function(_, onResponse) {
       respError = null;
 
+      if (stream) {
```

Comment:
@stephenplusplus since this is the first gRPC API that implemented streaming endpoints, I wasn't 100% how to move forward with this. My initial reaction is pretty much this:

If they only offer a streaming endpoint, then those specific APIs [in our lib] should only return streams and should not accept callbacks. Thoughts on all of this?